xml/System.Xml.Xsl/XslTransform.xml

```diff
@@ -1364,12 +1364,12 @@ Root node is book.
  `print_root.xsl`  
   
 ```xml  
-<style sheet version="1.0" xmlns="http://www.w3.org/1999/XSL/Transform" >  
+<stylesheet version="1.0" xmlns="http://www.w3.org/1999/XSL/Transform" >  
```

Comment:
I'd have to test this since it's been a while I don't deal with this but it might need some further refinements.
something like `<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" >`

Reply:
Yeah, I've never seen it without the namespace either, but presumed it was valid as that's how everything was written. I was guessing that `stylesheet`/`style sheet` had come from a dodgy find and replace!